mailer.rst

```diff
@@ -115,9 +116,9 @@ Service               Install with                                    Webhook su
 `SendGrid`_           ``composer require symfony/sendgrid-mailer``    yes
 ===================== =============================================== ===============
 
-.. versionadded:: 7.1
+.. versionadded:: 7.2
 
-    The Azure and Resend integrations were introduced in Symfony 7.1.
+    The Mailomat integration was introduced in Symfony 7.2.
```

Comment:
We need to keep 7.1 versionadded, please addd one for 7.2 below

Reply:
changed accordingly.